webpack.config.js

```diff
@@ -50,52 +50,74 @@ var config = [{
     './modules/data_integrity_flag/js/index.js': './modules/data_integrity_flag/jsx/index.js',
     './modules/imaging_uploader/js/index.js': './modules/imaging_uploader/jsx/index.js',
-    './modules/acknowledgements/js/columnFormatter.js': './modules/acknowledgements/jsx/columnFormatter.js'
+    './modules/acknowledgements/js/columnFormatter.js': './modules/acknowledgements/jsx/columnFormatter.js',
   },
   output: {
-    path: './',
-    filename: '[name]'
+    path: __dirname + '/',
+    filename: '[name]',
   },
   module: {
-    loaders: [
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader'],
+      },
       {
         test: /\.js$/,
         exclude: /node_modules/,
-        loader: 'babel-loader'
-      }
-    ]
+        use: ['babel-loader', 'eslint-loader'],
+      },
+      {
+        test: /\.json$/,
+        loader: 'json',
+      },
+    ],
   },
   resolve: {
-    root: path.resolve(__dirname),
     alias: {
-      util: 'htdocs/js/util',
-      jsx: 'jsx',
-      Breadcrumbs: 'jsx/Breadcrumbs',
-      DynamicDataTable: 'jsx/DynamicDataTable',
-      FilterForm: 'jsx/FilterForm',
-      Form: 'jsx/Form',
-      Markdown: 'jsx/Markdown',
-      PaginationLinks: 'jsx/PaginationLinks',
-      Panel: 'jsx/Panel',
-      ProgressBar: 'jsx/ProgressBar',
-      StaticDataTable: 'jsx/StaticDataTable',
-      Tabs: 'jsx/Tabs'
+      util: path.resolve(__dirname, './htdocs/js/util'),
+      jsx: path.resolve(__dirname, './jsx'),
+      Breadcrumbs: path.resolve(__dirname, './jsx/Breadcrumbs'),
+      DynamicDataTable: path.resolve(__dirname, './jsx/DynamicDataTable'),
+      FilterForm: path.resolve(__dirname, './jsx/FilterForm'),
+      Form: path.resolve(__dirname, './jsx/Form'),
+      Markdown: path.resolve(__dirname, './jsx/Markdown'),
+      PaginationLinks: path.resolve(__dirname, './jsx/PaginationLinks'),
+      Panel: path.resolve(__dirname, './jsx/Panel'),
+      ProgressBar: path.resolve(__dirname, './jsx/ProgressBar'),
+      StaticDataTable: path.resolve(__dirname, './jsx/StaticDataTable'),
+      Tabs: path.resolve(__dirname, './jsx/Tabs'),
     },
-    extensions: ['', '.js', '.jsx']
+    extensions: ['*', '.js', '.jsx', '.json'],
   },
   externals: {
-    react: 'React'
+    react: 'React',
   },
   node: {
-    fs: "empty"
+    fs: 'empty',
   },
   devtool: 'source-map',
-  plugins: [new webpack.optimize.UglifyJsPlugin({mangle: false})]
+  plugins: [],
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
+        cache: true,
+        parallel: true,
+        uglifyOptions: {
+          compress: false,
+          ecma: 6,
+          mangle: false,
+        },
+        sourceMap: true,
+      }),
+    ],
+  },
 }];
 
 // Support project overrides
 if (fs.existsSync('./project/webpack-project.config.js')) {
-  var projConfig = require('./project/webpack-project.config.js');
+  const projConfig = require('./project/webpack-project.config.js');
```

Comment:
@intralizee does the webpack project need to be updated too??

Reply:
I was actually confused about what to do there because I don't see a "webpack-project.config.js" file in my project directory. Currently I don't believe that line does or break anything.